docs/mp/health/01_introduction.adoc

```diff
@@ -26,68 +26,92 @@
-can monitor each service and take action, such as restarting a service instance if it has failed
-or temporarily shunting traffic away from the instance if the service is unable to process
-incoming requests normally.
 
-== About the MicroProfile Health Specification
-Helidon MP implements the MicroProfile Health
-link:http://download.eclipse.org/microprofile/microprofile-health-{health-release}/microprofile-health-spec.html[spec].
+== Usage
+
+Helidon MP Health implements MicroProfile Health Checks
+link:http://download.eclipse.org/microprofile/microprofile-health-{health-release}/microprofile-health-spec.pdf[Specification].
 The spec prescribes how external tools probe a service's health checks and how you
 implement health checks as part of your microservice that are specific to your service's needs.
 
-== Concepts
-
-=== Liveness, Readiness, and Startup Checks
+=== Concepts - Liveness, Readiness, and Startup Checks
 
 MicroProfile Health supports three types of health checks:
 
-_Liveness_ checks report whether the runtime environment in which the service is running
+* _Liveness_ checks report whether the runtime environment in which the service is running
 is sufficient to support the work the service performs.
 The environment is beyond the control of
 the service itself and typically cannot improve without outside intervention.
 If a microservice instance reports a `DOWN`
 liveness check, it should never report `UP` later.
 It will need to be stopped and a replacement instance created.
 
-_Readiness_ checks report whether the service is _currently_ capable of performing its work.
+* _Readiness_ checks report whether the service is _currently_ capable of performing its work.
 A service that reports `DOWN` for its readiness cannot _at the moment_ do its job, but at
 some future point it might become able to do so without requiring a restart.
 
-_Startup_ checks indicate whether the service has started to the point where liveness and readiness checks even make sense.
- A service reporting `DOWN` for a startup check is still initializing itself and normally will report `UP`, assuming it has started successfully.
+* _Startup_ checks indicate whether the service has started to the point where liveness and readiness checks even make sense.
+A service reporting `DOWN` for a startup check is still initializing itself and normally will report `UP` soon, assuming it is able to start successfully.
 
-The following table describes more about these types of health checks, including how an orchestrator
-such as Kubernetes might react.
 
-=== Known Health Check Endpoints
+== API
 A MicroProfile-compliant service reports its health via known REST endpoints. Helidon MP
-provides these endpoints automatically as part of every MP microservice.
+provides these endpoints automatically as part of every MP microservice that includes health support..
```

Comment:
This section is marked as API. API stands for application programming interface - I expect to see classes, javadoc, interfaces. 
I do not think the content here can be classified as API

Reply:
I would name this section "REST Endpoints".

Reply:
Done)